bridge-token-factory/src/lib.rs

```diff
@@ -164,7 +223,7 @@ impl BridgeTokenFactory {
             proof_1,
             &env::current_account_id(),
             env::attached_deposit(),
-            FINISH_DEPOSIT_GAS + MINT_GAS,
+            FINISH_DEPOSIT_GAS + MINT_GAS + FT_TRANSFER_CALL_GAS,
```

Comment:
We probably only need to charge `FT_TRANSFER_CALL_GAS` if `ft_transfer_call` is going to be used.

Reply:
https://github.com/aurora-is-near/rainbow-token-connector/issues/64